ArcGISRuntimeSDKQt_CppSamples/CloudAndPortal/TokenAuthentication/TokenAuthentication.h

```diff
@@ -38,11 +40,14 @@ class TokenAuthentication : public QQuickItem
 
   void componentComplete() override;
   static void init();
-  Q_INVOKABLE void loadSecuredLayer();
 
 private:
+  QUrl m_portalURL = QUrl("https://www.arcgis.com/");
```

Comment:
optional. these could be moved to the cpp file in an anonymous namespace rather than having them as state for the class.

Reply:
Agreed, this can just be set in the .cpp file. Furthermore, I'm not actually certain it needs to be a variable because it's only referenced once. Also arcgis.com is the default portal url so it really doesn't need to be passed at all.